docs/api/ref/api.yml

```diff
@@ -430,6 +466,16 @@ components:
       name: lc
       required: false
       description: '2 letter code of the language of the user. Used for localizing some fields in returned values (e.g. knowledge panels). If not passed, the language may be inferred by the Accept-Language header of the request.'
+    # same as lc above but within a path (using path template)
+    lc_in_path:
+      schema:
+        type: string
+        example: 'fr'
+      in: path
+      name: lc
+      required: true
+      description: '2 letter code of the language of the user. Used for localizing some fields in returned values (e.g. knowledge panels). If not passed, the language may be inferred by the Accept-Language header of the request.'
```

Comment:
If lc is not passed, it is inferred by the subdomain. There seems to be a caching bug though as I  get different results for the same request to https://fr.openfoodfacts.org/api/v2/attribute_groups

Reply:
I'm fixing the docs to use the short URL instead.